tests/foreman/cli/test_remoteexecution.py

```diff
@@ -341,15 +313,15 @@ def test_positive_run_default_job_template(self):
             'search-query': "name ~ {0}".format(self.client.hostname),
         })
         self.assertEqual(
-                invocation_command['success'],
-                u'1',
-                'host output: {0}'.format(
-                    ' '.join(JobInvocation.get_output({
-                        'id': invocation_command[u'id'],
-                        'host': self.client.hostname})
-                    )
+            invocation_command['success'],
+            u'1',
+            'host output: {0}'.format(
+                ' '.join(JobInvocation.get_output({
```

Comment:
Speaking about performance improvements:

Let's say i have such example test:
```python
import time


def heavy_calculation():
    print "i'm wasting a lot of processor time no matter if i'm needed at all"
    time.sleep(10)
    return 'some output'


class FooTestCase(CLITestCase):

    def test_foo(self):
        self.assertEqual(True, True, '{}'.format(heavy_calculation()))
```
You've probably got the the result of executing such test, don't you? 🙂 
```
test_contentview.py 2017-10-26 12:11:40 - robottelo - INFO - Started setUpClass: tests.foreman.cli.test_contentview/FooTestCase
2017-10-26 12:11:40 - robottelo - DEBUG - Started Test: FooTestCase/test_foo
i'm wasting a lot of processor time no matter if i'm needed at all
2017-10-26 12:11:50 - robottelo - DEBUG - Finished Test: FooTestCase/test_foo
.2017-10-26 12:11:50 - robottelo - INFO - Started tearDownClass: tests.foreman.cli.test_contentview/FooTestCase
```
We have 18 (!) tests in this module with bad assertions design which is causing CLI commands execution when assertion is passing and they aren't needed at all. Here's a good candidate for refactor

Reply:
got it, pls raise an issue for this, and i hope @pondrejk will be able to rework this.

Reply:
whoa, a little backfire :) filed #5463  to find a different solution